ParticleObjects/particle_class.f90

```diff
@@ -55,6 +55,7 @@ module particle_class
     integer(shortInt)          :: matIdx   = -1     ! Material index where particle is
     integer(shortInt)          :: cellIdx  = -1     ! Cell idx at the lowest coord level
     integer(shortInt)          :: uniqueID = -1     ! Unique id at the lowest coord level
+    integer(shortInt)          :: splitCount = 0    ! Counter of number of splits
```

Comment:
It may be a silly question, but doe we use the `splitCount` in the `particleState`? 
Sorry, I just get paranoid lately that more and more data gets into `particleState` and it just grows on...
I guess it will become necessary if the particles inherit the `splitCount` from their primogenitor, but otherwise maybe we need `spliCount` only in the particle?

Reply:
No, that's fair. I think I added it just in case we wanted to pursue the 'family' weight thing, in which case it could have been useful, but it can surely be avoided for the moment.